.github/workflows/master.yml

```diff
@@ -15,9 +15,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v1
-      - name: Set up JDK 1.8
+      - name: Set up Java 11
         uses: actions/setup-java@v1
         with:
-          java-version: '8.0.282'
+          java-version: '11'
```

Comment:
please specify the Java version more strict here, like we did with 8

Reply:
I agree